### PR TITLE
Adds @readonly to the docs

### PR DESCRIPTION
**Pull request by Ascenio**
Also updated the summary to simplify searching for `@foo_bar`. Let me know if you don't want this behavior.

### CODE REVIEW

**Comment by pavanpodila** [comment]
We need it on the docs for website, not just readme

**Comment by Ascenio** [comment]
Under which section should I add it? Maybe under `Core Concepts` with the same body?

**Comment by pavanpodila** [comment]
Put under api

**Comment by Ascenio** [comment]
But then should I modify the body of the README to be simpler? I can only think of a before and after comparison to make it shorter.

**Comment by pavanpodila** [comment]
Yes. Readme cannot be long. Keep the details, examples in the Api page

**Review by pavanpodila — CHANGES_REQUESTED** [review]
I think we should shorten the description of read-only in readme. Also move it further down in order

**Comment by Ascenio** [comment]
What about the order? Should it be bellow computed but still under observables' section?

File: README.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ Language: [English](README.md) | [Português](translation/pt-BR/README.md) | [Ch
 - **[Introduction](#introduction)**
 - **[Core Concepts](#core-concepts)**
   - [Observables](#observables)
-  - [Computed Observables](#computed-observables)
+    - [@observable](#observables)
+    - [@readonly](#readonly)
```
Comment (pavanpodila) [inline README.md:31]: I suggest removing from readme as this detail is not needed at this level

File: README.md [hunk 2]
```diff
@@ -133,6 +135,44 @@ will fade away and you will mostly focus on the code within the braces.
 
 **Note**: Annotations are available via the **[mobx_codegen](https://github.com/mobxjs/mobx.dart/tree/master/mobx_codegen)** package.
 
+### Readonly
```
Comment (pavanpodila) [inline README.md:138]: all of this can be move to the `observables.mdx`

**Review by pavanpodila — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Ascenio) [reply README.md:138]: I have already moved the previous README to `observables.mdx`. Should I just remove the readonly from the README?

Comment (pavanpodila) [reply README.md:138]: Ya let's remove from readme

Comment (Ascenio) [reply README.md]: Done

**Comment by Ascenio** [comment]
@pavanpodila I think it's weird to leave the reference out of the README because users who don't access the website won't know it exists. Anyway, the references to the site have been added and the ones from the README have been removed already.

**Comment by pavanpodila** [comment]
We can keep 2-3 lines about read only in the main Readme.

**Comment by Ascenio** [comment]
I've simplified both READMEs and left the detailed explanation to the site.

**Review by pavanpodila — APPROVED** [review]

**Pull request merged**